src/Simulation/CsharpGeneration.App/Program.fs

```diff
@@ -49,26 +48,25 @@ type Logger() =
 let generateFiles (options : Options) = 
     let logger = new Logger()
     let outputFolder = if String.IsNullOrWhiteSpace options.QSTFileName then null else options.OutputFolder
+    let codeGenDll = typeof<Emitter>.Assembly.Location
+    let assemblyConstants = new Dictionary<string, string>()
+    assemblyConstants.[Emitter.SupportUnitTests] <- "true" // always permit unit tests
```

Comment:
We could remove it, but the idea was to have a handle that would permit to e.g. check if the project references the necessary xUnit packages and only support unit tests if it does (rather than failing). Just failing the C# compilation in this case is the alternative.

Reply:
removed